utils.groovy

```diff
@@ -25,4 +25,35 @@ void addRemoteRef(String branchName) {
   }
 }
 
+/**
+  * Get Version
+  */
+def getVersion() {
```

Comment:
Technically this function is not being used yet, but it was cherry-picked along with the other version changes. I wanted to keep changes to the `Jenkinsfile` minimal since those files are still being refactored as a part of rstudio/rstudio-pro#4140